components/cmake/modules/FindPIO.cmake

```diff
@@ -0,0 +1,24 @@
+# - Try to find PIO
+#
+# Placeholder code until spio gets a proper config.cmake
+#
+# Once done, this will define:
+#
+#   The "spio" target
+#
+
+if (NOT PIO_LIBDIR)
+  set(PIO_LIBDIR "${INSTALL_SHAREDPATH}/lib")
+endif()
+
+if (PIO_VERSION STREQUAL 2)
+  # This is a pio2 library
+  set(PIOLIBS "${PIO_LIBDIR}/libpiof.a;${PIO_LIBDIR}/libpioc.a")
+else()
+  # This is a pio1 library
+  set(PIOLIBS "${PIO_LIBDIR}/libpio.a")
```

Comment:
Does E3SM hard-code static libraries?

Reply:
This will go away once scorpio has a good cmake config.